src/release/extensions/css/SCALA-LICENSE.txt

```diff
@@ -1,35 +0,0 @@
-SCALA LICENSE
```

Comment:
I see this was deleted, but didn't see a replacement. Any reason why there was no replacement?

Reply:
This extension is no longer based on scala- Andrea rewrote it some years ago to be pure java.